rexfw/statistics/writers/graphite.py

```diff
@@ -0,0 +1,74 @@
+from pickle import dumps
+from struct import pack
+from socket import socket
+
+# timestamp is the number of seconds since unix epoch time.
+# Carbon-cache will use the time of arrival if the timestamp is set to -1.
+# https://graphite.readthedocs.io/en/stable/feeding-carbon.html
+
+
+def _make_tuples_for_stepsize(job_name, elements):
+    listOfStatsTuples = []
+    elements = [e for e in elements if e.name == "stepsize"]
+    for e in elements:
+        timestamp = -1
+        listOfStatsTuples.append((job_name + ".stepsize", (timestamp, e.current_value)))
+    return listOfStatsTuples
+
+
+def _make_tuples_for_acceptance_rates(job_name, elements):
+    listOfStatsTuples = []
+    elements = [e for e in elements if e.name == "acceptance rate"]
+    for e in elements:
+        timestamp = -1
+        if len(e.origins) == 1:
+            listOfStatsTuples.append(
+                (
+                    job_name + ".acceptance_rate.local",
```

Comment:
It would be great to have the string in `origins` as part of the Graphite path, so that we know to which replica the acceptance rate / time step belongs.